src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -186,6 +186,7 @@ $sidenav-icon-padding-size: 5px;
 
 .sidenav-toggle-wrapper {
   display: flex;
+  margin-right: $nav-padding / 2;
```

Comment:
Ah whoops. I was trying to simplify it on purpose, because I didn't think the margin would impact the alignment of the centered title at S/M breakpoints (but it looks like it does slightly)

Reply:
8fad694a909e970aa483277ebfebb5ac8b1542bb